docs/static-header-footer/bootstrap-3.html

```diff
@@ -210,7 +210,7 @@ <h2 class="wb-inv">About this site</h2>
 						<li><a href="https://www.canada.ca/en/transparency/privacy.html">Privacy</a></li>
 					</ul>
 				</nav>
-				<div class="col-xs-6 visible-sm visible-xs tofpg"> <a href="#wb-cont">Top of Page <span class="glyphicon glyphicon-chevron-up"></span></a> </div>
+				<div class="col-xs-6 visible-sm visible-xs tofpg"> <a href="#wb-tphp">Top of Page <span class="glyphicon glyphicon-chevron-up"></span></a> </div>
```

Comment:
The updated link destination would result in a broken link in this page since it lacks skip navigation links (the ``wb-tphp`` ID is usually set on the skipnav links' ``<ul>`` element).

@GormFrank Is that intentional? Should the skipnav links be re-added to this page?